src/routes/api/index.js

```diff
@@ -1,10 +1,12 @@
 import express from 'express';
 
 import authRouter from './auth.router';
+import requestRouter from './requests.router';
 
 
 const router = express.Router();
 
 router.use('/auth', authRouter);
```

Comment:
You can chain routes like so:

```
app.use(authRouter, requestRouter)

```
this will save out app from being bloated with routes.

Reply:
We did this earlier together...if you remembered and someone changed it back to this. In order to avoid back and forth, I will like to leave it this way until we have our meeting over the weekend as this is not really a bug. I hope you accept this case. Thanks again for your feedback

Reply:
Okay. I was not exactly talking about this though. This can be extended. But it's fine. We will refactor.